config/initializers/audit.rb

```diff
@@ -5,3 +5,10 @@
 
 Audit = Audited.audit_class
 Audit.send(:include, AuditExtensions)
+
+# if this is a migration rake task we disable auditing in particular models
+if Foreman.in_rake?('db:migrate')
+  [ Host::Managed, Hostgroup, ProvisioningTemplate, Ptable, User ].each do |klass|
```

Comment:
Ah yes, this would work, since the class is loaded, audited is called and then we change it back after.  Sorry, I thought it wasn't working at the moment.

My last 2c on the method chaining is that you could chain it in AR::Base after the audited gem has loaded, rather than chaining their concern - as long as it's always called `audited`, then you'll be able to chain it, no matter where it was loaded from, I think.